### ISSUE TITLE
Improve Polar support on Windows Surface Tablets

### ISSUE THREAD

**Issue by burtonator** [burtonator/polar-bookshelf#473]
Tablet support would rock but it's not super easy... need to keep running notes here and add features as they arise.

**Comment by jullimullins** [comment]
Biggest improvement would be highlighting with pen or touch screen without clicking anything.  The same way like Adobe Reader or Foxit MobilePDF works when annotation is activated. The topic also could be changed to windows 10 laptops with touch screen and/or pen support.

**Comment by jullimullins** [comment]
Polar Bookshelf is built on Node + Chromium, so I'm just wondering if there is similar setting 
as in chrome some where:
![image](https://user-images.githubusercontent.com/16438266/50381565-0f68db00-0693-11e9-9516-1b3380d2a101.png)
After enabling this in chrome, the selection of a text with stylus is a lot easier.

**Comment by burtonator** [comment]
How do you enable it in chrome? This is one of the flags?

**Comment by burtonator** [comment]
yeah.. looks like it's one of the flags. I'm not sure if Electron supports this flag. It takes command line arguments but this one is definitely not documented that I could fine.

I'm wondering if we can just pass ANY chrome flag?

there's a huge list:

https://peter.sh/experiments/chromium-command-line-switches/

just not sure which ones are supported.

**Comment by jullimullins** [comment]
Yes, it's a flag. I went through the switches list but could not find similar one.
